installation/developer-installation/full-developer-installation.md

```diff
@@ -0,0 +1,247 @@
+---
+description: Setting up the recommended OctoBot development environment
+---
+
+# full developer installation
+
+{% hint style="info" %}
+This page is referring to OctoBot in versions superior to 0.4.0.
+{% endhint %}
+
+* ****[**Install OctoBot requirements**](https://app.gitbook.com/s/XGhS0OJcF37WFUAtCJrB/c/K2S34tUZ8NdXf7pYP47C/guides/dev-environment-setup#install-octobot-requirements)****
```

Comment:
I don't think we should keep these links. You should have links using docs.octobot.online or just relative links